src/rgw/rgw_civetweb.cc

```diff
@@ -67,17 +67,25 @@ size_t RGWCivetWeb::complete_request()
   return 0;
 }
 
-void RGWCivetWeb::init_env(CephContext *cct)
+[[nodiscard]] int RGWCivetWeb::init_env(CephContext *cct)
```

Comment:
@theanalyst shouldn't you have the `[[nodiscard]]` in the class definition?

Reply:
ack